test/data/ais-cpa-tcpa.json

```diff
@@ -0,0 +1,24 @@
+{
+  "vessels": {
+    "urn:mrn:signalk:uuid:c0d79334-4e25-4245-8892-54e8ccc8021d": {
+      "uuid": "urn:mrn:signalk:uuid:c0d79334-4e25-4245-8892-54e8ccc8021d",
+      "navigation": {
+        "courseOverGroundTrue": {
+          "value": 206.9,
+          "timestamp": "2014-08-15T18:00:00.755Z",
+          "pgn": 130577,
+          "$source": "sourceLabel.160"
+        }
+      },
+      "navigation": {
```

Comment:
Agree. I tried to keep to the CPA and TCPA nomenclature, but this is cleaner and more understandable. Test file fixed